### PR TITLE
0.18: add contributors from git log, polish release notes

### PR DESCRIPTION
**Pull request by guggero**
Adds contributors found through `git shortlog -sne v0.17.5-beta..v0.18.0-beta.rc1` (with some manual work to arrive at GitHub usernames or name/surname if available).

Also polishes the release notes somewhat.

### CODE REVIEW

File: docs/release-notes/release-notes-0.18.0.md [hunk 1]
```diff
@@ -118,20 +119,17 @@
...
 
-* [Fixed](https://github.com/lightningnetwork/lnd/pull/8545) utxo selection
+* [Fixed](https://github.com/lightningnetwork/lnd/pull/8545) UTXO selection
   for the internal channel funding flow (Single and Batch Funding Flow). Now
-  utxos which are unconfirmed and originated from the sweeper subsystem are not
+  UTXOs which are unconfirmed and originated from the sweeper subsystem are not
   selected because they bear the risk of being replaced (BIP 125 RBF).
 
-* [Fixed](https://github.com/lightningnetwork/lnd/pull/8685) lncli "bumpfee"
+* [Fixed](https://github.com/lightningnetwork/lnd/pull/8685) `lncli bumpfee`
```
Comment (yyforyongyu) [inline docs/release-notes/release-notes-0.18.0.md:128]: nit: feel this should be removed since this bug was introduced in this release and later fixed.

**Review by yyforyongyu — APPROVED** [review]
Nice!

**Review by ellemouton — APPROVED** [review]
⭐

File: docs/release-notes/release-notes-0.18.0.md (hunk 1 above)
Comment (guggero) [reply docs/release-notes/release-notes-0.18.0.md]: Done!

File: docs/release-notes/release-notes-0.18.0.md [hunk 2]
```diff
@@ -210,8 +205,9 @@
   `lnd.conf`](https://github.com/lightningnetwork/lnd/pull/8310)
   for the `rpcuser` and `rpcpass` fields to better protect the secrets.
 
-* When computing a minimum fee for transaction construction, `lnd` [now takes our
-bitcoin peers' feefilter values into account](https://github.com/lightningnetwork/lnd/pull/8418).
+* When computing a minimum fee for transaction construction, `lnd` [now takes
+  our bitcoin peers' `feefilter` values into
```
Comment (saubyk) [inline docs/release-notes/release-notes-0.18.0.md:209]: nit: should be 'its' instead of 'our'

**Pull request merged**